src/components/list/list.js

```diff
@@ -129,6 +129,24 @@ function mdListItemDirective($mdAria, $mdConstant, $mdUtil, $timeout) {
         }
       }
 
+      function setupProxiedMenu() {
+        var menuEl = angular.element(proxyElement);
+        var isLeftAligned = proxyElement.parentNode.firstElementChild == proxyElement;
+
+        // Set the position mode of the proxied menu
+        menuEl.attr('md-position-mode', (isLeftAligned ? 'target target' : 'target-right target'));
+
+        // Apply menu open binding to menu button
+        var menuOpenButton = menuEl.children().eq(0);
```

Comment:
This method is used by the original menu too ([See here](https://github.com/angular/material/blob/master/src/components/menu/js/menuDirective.js#L162)). I already had `find`, and validating the right parent. I think using the first children is the right way to get the triggering element.